contracts/peg/SimpleFeiDaiPSM.sol

```diff
@@ -33,10 +33,9 @@ contract SimpleFeiDaiPSM {
     function mint(
         address to,
         uint256 amountIn,
-        uint256 minAmountOut
+        uint256 // minAmountOut
```

Comment:
If fei protocol is entering a terminal state where it will no longer be able to be upgraded, why even have a mint function?

Reply:
Fei will be a DAI wrapper. Needs to be mintable and redeemable to prevent any issues with the peg breaking